app/views/NewIncomeSources.scala.html

```diff
@@ -0,0 +1,131 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import _root_.implicits.ImplicitDateFormatterImpl
+@import views.html.layouts.unifiedLayout
+@import views.html.components._
+@import exceptions.MissingFieldException
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import models.incomeSourceDetails.BusinessDetailsModel
+@import models.incomeSourceDetails.PropertyDetailsModel
+
+@this(
+    mainTemplate: unifiedLayout,
+    implicitDateFormatter: ImplicitDateFormatterImpl,
+    h1: h1,
+    h2: h2,
+    p: p,
+    link: link,
+    hr: hr,
+    govukTable: GovukTable
+)
+
+@(
+    soleTraderBusinesses: List[BusinessDetailsModel],
```

Comment:
* pls create a required view model and wrap this into these field in it;
* as originally discussed, we expect a method in incomeSourceDetailsService to return this model;